src/test/java/com/issoft/training/tests/RemoteDriverLoginTests.java

```diff
@@ -0,0 +1,61 @@
+package com.issoft.training.tests;
+
+import com.issoft.training.config.ConfProperties;
+import com.issoft.training.listener.AllureListener;
+import com.issoft.training.pages.LoginAccountPage;
+import com.issoft.training.utility.Constants;
+import io.qameta.allure.*;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import static com.issoft.training.base.TestBase.getScreenShotFile;
+import static com.issoft.training.config.ConfProperties.getProperty;
+import static com.issoft.training.webdrivers.RemoteDriver.launchingRemoteDriver;
+
+@Listeners({ AllureListener.class })
+@Epic("Regression tests")
+@Feature(value = "Automation practice - Login Account Remote Tests")
+public class RemoteDriverLoginTests {
+    private WebDriver driver;
+    LoginAccountPage loginAccountPage;
+    private static final String SAUCE_LABS_USERNAME = "sergeimaleyko";
+    private static final String SAUCE_LABS_ACCESS_KEY = "4dc55c92-bccd-400d-993e-dad87507bff8";
+    private static final String URL = "https://"+ SAUCE_LABS_USERNAME + ":" + SAUCE_LABS_ACCESS_KEY + "@ondemand.eu-central-1.saucelabs.com:443/wd/hub";
+    private static final String SELENOID_LOCALHOST_HUB = "http://localhost:4444/wd/hub";
+
+    @BeforeTest(groups = { "saucelabsTests", "gridTests" })
+    @Parameters({"webdriver", "browser", "version", "platform", "test_name"})
+    public void setup(String wd, String br, String vr, String pf, String tn) {
+        if(wd.equalsIgnoreCase("saucelabs")){
+            driver = launchingRemoteDriver(br, vr, pf, tn, URL); }
+        else if (wd.equalsIgnoreCase("grid")){
+            driver = launchingRemoteDriver(br, null, null, null, SELENOID_LOCALHOST_HUB); }
+    }
```

Comment:
It shouldn't be a part of beforeTest. It should be moved to class where you do configuration of the WebDriver.
Usually it happens that all tests are executed remotely(or locally). So it more global configuration of your framework than configuration on the test class. Configuration on test class cannot be reused by other test class.
In such case please move those configuration to your WebDriver class. 

Also I don't see a reason to have such tests. Do I understand correctly that it check that login tests works with remote configuration? If yes, no need to create test(s) which check(s) WebDriver configuration

Reply:
Moved a driver logic to WebDriver class.